docs/document/content/user-manual/shardingsphere-scaling/usage.cn.md

```diff
@@ -289,3 +284,225 @@ mysql> preview select count(1) from t_order;
 ### DistSQL 手动模式接口
 
 数据校验、切换配置等操作可以手动执行。详情请参见：[RAL #弹性伸缩](/cn/user-manual/shardingsphere-proxy/distsql/syntax/ral/#%E5%BC%B9%E6%80%A7%E4%BC%B8%E7%BC%A9)。
+
+### DistSQL 手动模式完整流程示例
+
+本示例演示从已有 MySQL 数据库迁移到 proxy。
+
+除了明确说明在 MySQL 执行的 SQL，其他都是在 proxy 执行。
+
+#### 新建源端库
+
+已有数据不需要这个步骤。这里是模拟一个源端库用于测试。
+
+在 MySQL 执行 SQL：
+```sql
+drop database if exists scaling_ds_0;
```

Comment:
Do keywords need to be capitalized?

Reply:
All SQL keywords added this time are changed to uppercase.